tests/integration/secretsmanager/test_secretsmanager.snapshot.json

```diff
@@ -1,3130 +0,0 @@
-{
```

Comment:
I think this file was deleted by accident @MEPalma? Still seems to be used by the related tests.

Reply:
Indeed @steffyP thank you